packages/cfpb-layout/src/organisms/wells.less

```
@@ -19,7 +19,7 @@
         color: @white;
 
         a {
-            // default, `:visited`, `:hover`, `:focus`, and `:active` states in the first five arguments)
+            // default, `:visited`, `:hover`, `:focus`, and `:active` states are the first five arguments.
```

```suggestion
            // Arguments: default, `:visited`, `:hover`, `:focus`, and `:active` states.
```

Just to make it consistent with the other lines.